src/shared/components/plots/WaterfallPlot.tsx

```diff
@@ -454,11 +461,18 @@ export default class WaterfallPlot<D extends IBaseWaterfallPlotData> extends Rea
     
     @bind
     private getChart() {
+        const svgViewBox = () => {
+            if (this.props.horizontal) {
+                return `0 -10 ${this.svgWidth} ${this.svgHeight}`;
+            }
+            return `0 0 ${this.svgWidth} ${this.svgHeight}`
+        };
```

Comment:
Better move this out of the getCart() method and change into a @computed field that depends on this.props.horizontal?